mackerel-plugin-multicore/lib/multicore.go

```diff
@@ -120,17 +121,17 @@ func parseProcStat(out io.Reader) (map[string]procStats, error) {
 			continue
 		}
 
-		floatValues, err := parseFloats(values)
+		counterValues, err := parseCounters(values)
 		if err != nil {
 			return nil, err
 		}
 
-		total := 0.0
-		for _, v := range floatValues {
+		var total uint64
+		for _, v := range counterValues {
 			total += v
```

Comment:
Is it necessary to consider overflows?

Reply:
Basically it doesn't matter since we take the difference of total cpu counters. So, should we skip the `last.Total > current.Total` check? I think `current.Total - last.Total` is valid even if `total += v` overflows since the subtraction of uint64 results in the actual difference (strictly speaking, the calculation fails when the cpu counter overflows twice in the duration, but it may not occur in real situation).